content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md

```diff
@@ -432,6 +432,7 @@ Istio supports reading a few different Secret formats, to support integration wi
 * A TLS Secret with keys `tls.key` and `tls.crt`, as described above. For mutual TLS, a `ca.crt` key can be used.
 * A generic Secret with keys `key` and `cert`. For mutual TLS, a `cacert` key can be used.
 * A generic Secret with keys `key` and `cert`. For mutual TLS, a separate generic Secret named `<secret>-cacert`, with a `cacert` key. For example, `httpbin-credential` has `key` and `cert`, and `httpbin-credential-cacert` has `cacert`.
+* The `cacert` entry can be a CA bundle: concatenated individual CA certificates.
```

Comment:
Maybe, "The `cacert` entry can be a CA bundle consisting of concatenated individual CA certificates."

Reply:
Thanks, changed. added "value" for clarity too but maybe it should be

`cacert` key's value can be...

I put so far

`cacert` key value can be...

problem with english as a second language, is I'm not sure, lmk

Reply:
nit: can we clarify that these are "individual Root CA certificates"

Reply:
It should work with any CA, why wouldn't it for intermediate vs root? as long as envoy can verify the signer

Reply:
agreed, but it would it be a good idea to add a test to "tests/integration/security/sds_ingress/" before we claim support for this in a release-branch? Not aware of any other scenarios where we pass an intermediate cert to envoy as a TLS trust anchor; please enlighten me if you know of any

Reply:
here is an unintegrated test (the test I used to check that indeed it's working and CAs can be changed without errors):

https://github.com/istio/istio/pull/33880/files

Reply:
I tested with intermediate CAs and that works as well but you need to put the whole chain in the cacert (ie both the intermediate and the root) - see the MR above for the test

Reply:
LGTM to make change